client/src/components/JobDescription.jsx

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from "react";
+import { Grid, Typography, Link } from "@mui/material";
+import IconButton from "@mui/material/IconButton";
+import KeyboardArrowUpOutlinedIcon from "@mui/icons-material/KeyboardArrowUpOutlined";
+import { Link as RouterLink } from "react-router-dom";
+
+const JobDescription = (props) => {
+  const [readMore, setReadMore] = useState(false);
+  const toggle = () => {
```

Comment:
setReadMore can also take a callback as an argument- it looks like this

Check out the section that starts with "To solve this problem, you may pass an updater function to setAge instead of the next state"

https://react.dev/reference/react/useState